schema/home/params.js

```diff
@@ -0,0 +1,24 @@
+import {
+  GraphQLObjectType,
+  GraphQLString,
+} from 'graphql';
+
+const HomePageModuleParams = new GraphQLObjectType({
+  name: 'HomePageModulesParams',
+  fields: {
+    gene_id: {
+      type: GraphQLString,
+    },
+    medium: {
+      type: GraphQLString,
+    },
+    price_range: {
+      type: GraphQLString,
+    },
+  },
+});
+
+export default {
+  type: HomePageModuleParams,
+  resolve: ({ params }) => params,
+};
```

Comment:
So, in addition to populating generic gene's we need to return the params of the randomly chosen generic gene (so that we can fetch the same results after the initial page load).